code/game/gamemodes/cult/talisman.dm

```diff
@@ -270,8 +270,13 @@
 	var/mob/living/carbon/human/H = user
 	user.visible_message("<span class='warning'>Otherworldly armor suddenly appears on [user]!</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, arming yourself!</span>")
-	H.equip_or_collect(new /obj/item/clothing/suit/hooded/cultrobes/alt(user), slot_wear_suit)
-	H.equip_or_collect(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	if(!H.wear_suit) //Only get the equipment if the slot is free.
+		H.equip_or_collect(new /obj/item/clothing/suit/hooded/cultrobes/alt(user), slot_wear_suit)
+	if(!H.back)
+		H.equip_or_collect(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	if(!H.shoes)
+		if(H.has_organ_for_slot(slot_shoes)) //Might be empty for a reason.

```

Comment:
You'd still want the shoes if your foot got blown off; you can always get the foot re-attached later).